lib/bundle_notification.rb

```diff
@@ -1,7 +1,14 @@
 # frozen_string_literal: true
+
 require 'bundle_notification/mailer_helper'
 require 'bundle_notification/version'
 
 module BundleNotification
-  # Your code goes here...
+  def self.config
+    if block_given?
+      @config = Config.new
+      yield(@config)
+    end
+    @config ||= Config.new
+  end
```

Comment:
How about 2 methods: `configure` which always expects a block and `config` which just returns the current value? Looks like you put 2 methods into one just to have less of them 🙂